src/passes/LowerCase.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2015 WebAssembly Community Group participants
```

Comment:
The guidance changes on this, but the latest I've gotten is that the file date should be the one that the file was created on (it used to be: update every time a substantial modification happens, with date ranges).

This should all get fixed when / if we re-license :-)